auxiliary/BuybackSnx/storage.dump.sol

```diff
@@ -1,114 +0,0 @@
-// SPDX-License-Identifier: UNLICENSED
```

Comment:
dump lost?

Reply:
Not lost, they are being replaced by `storage.dump.json` files, have the advantage on being much easier to parse/use, ofc on the cost of readability but I think is a good trade off here. 
<img width="255" alt="Screenshot 2024-07-02 at 18 19 49" src="https://github.com/Synthetixio/synthetix-v3/assets/558901/5ddf428e-6bb4-4fb8-b4d1-f8ee8dc8f5c8">

Reply:
Also, this new implementation now stores in the dump the `size`, `slot` and `offset` for each variable, which point to the exact location on storage and it is what is used for validations.

Reply:
Like for example 

![image](https://github.com/Synthetixio/synthetix-v3/assets/28145325/f3da5df3-34b7-435f-bc44-c57bcf32527e)

Reply:
That's right, it is because on the previous version we were also dumping constants/immutables, but as they are not really required it is not generated anymore. Also, it dumped all the imported dependencies, regardless they were used or not. Now it just dumps local contracts that have `struct`'s defined, with the explicitly used imported values used by that struct.

- `utils/sample-project`: forgot to create the new dump for this one, going to add it.
- `auxiliary/BuybackSnx`: only had `immutable` constants, not storage
- `auxiliary/ERC4626ToAssetsRatioOracle`: only had `immutable` constants, not storage
- `auxiliary/SpotMarketOracle`: only had `immutable` constants, not storage
- `auxiliary/WstEthToStEthRatioOracle`: only had `immutable` constants, not storage
- `utils/core-modules`: this one didn't have the storage check task, which i think is ok as it is being checked by the used ones.